aten/src/ATen/native/transformers/cuda/sdp_utils.h

```diff
@@ -179,6 +204,9 @@ inline SDPBackend select_sdp_backend(sdp_params kernel_params) {
   if (use_flash_attention(kernel_params, !print_debug)) {
     return SDPBackend::flash_attention;
   }
+  if (use_mem_efficient_attention(kernel_params, !print_debug)){
```

Comment:
Problem with this design is how print debug statements. This works because the constraints are strictly less than for mem_effic than flash but if we add another kernel this doesn't make sense.

Reply:
We could have `forceUseSDPFlash` etc. flags that print debug statements if the branch wasn't taken.